cylc/rose/entry_points.py

```diff
@@ -215,8 +211,7 @@ def record_cylc_install_options(
     dumper.dump(cli_config, str(conf_filepath))
 
     # Merge the opts section of the rose-suite.conf with those set by CLI:
-    if not rose_conf_filepath.is_file():
```

Comment:
There is a check for this right at the beginning of the plugin's work. We shouldn't need to check again here.

Reply:
`touch` doesn't fail if the file is present so skipping the check is ok.